recipes/ibm_db_django/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "ibm_db_django" %}
+{% set version = "1.4.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9b7e504d84a51760729e16a7afc4942c535a05e1f775f96cbd33fb217e677a4e
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - sysroot_linux-64 2.17  # [linux64]
```

Comment:
Is this really needed?
```suggestion
```

Reply:
This is needed in order to build the package in linux platforms.

Reply:
Can you explain why? I don't think I've come across a `noarch: python` package which requires glibc 2.17 instead of glibc 2.12.

Reply:
With respect to this what my observation is that the building of the package on linux system fails when the GLIBC version is >2.17.
Hence we try to include the sysroot_linux-64 version as 2.17 dependency.
However we were not using ```noarch: python``` in such cases.
I am removing the sysroot_linux-64 dependency top check if the build on all of three platforms goes through.

Reply:
@chrisburr the change suggested is working fine in all platforms and the build is successful in linux mac and windows.